src/components/App.tsx

```diff
@@ -231,7 +231,7 @@ export default function App() {
             getPoints(canvas.curTime);
         } else {
             clearTimeout(loadingTimeout);
-            setIsLoadingPoints(false);
+            // setIsLoadingPoints(false);
```

Comment:
I removed this line. Because when opening the App, this UseEffect runs three times before the TrackManager is loaded. If `IsLoading` is set to false in the first run, it will not appear later when it should be there

Reply:
If you're going to leave this commented out - I would add some kind of comment. Otherwise I'd just delete this.